CrewChiefV4/App.config

```diff
@@ -100,14 +100,11 @@
       <setting name="custom_device_guid" serializeAs="String">
         <value />
       </setting>
-      <setting name="r3e_launch_params" serializeAs="String">
```

Comment:
i think I understand what you've done here, but I think we need to be able to specify launch params as well as the steam ID - some users need the -cefNoGpu or -noReplay params, as well as the steam app ID